src/Aquifer.Data/Entities/VersificationExclusionEntity.cs

```diff
@@ -0,0 +1,17 @@
+using Aquifer.Data.EventHandlers;
+
+namespace Aquifer.Data.Entities;
+
+public class VersificationExclusionEntity : IHasUpdatedTimestamp
+{
+    public int Id { get; set; }
```

Comment:
I don't think there's ever a situation where we look this up by `Id`. I'd just make a dual primary key on `BibleId` and `BibleVerseId`.